### PR TITLE
Adding space as per #10484

### PR DESCRIPTION
**Pull request by arafatkatze**

### CODE REVIEW

**Comment by aktech** [comment]
See #10514

**Comment by Arafatk** [comment]
@aktech  I am closing this.

**Comment by aktech** [comment]
If you want, you can edit this PR & I will merge this one.

**Comment by Arafatk** [comment]
@aktech  Thanks, 
 Can you tell me what corrections should be made?

File: doc/src/tutorial/gotchas.rst [hunk 1]
```diff
@@ -246,7 +246,7 @@ object.
 
 .. note::
 
-   On running the example above in sympy live,(1+1) is wrapped
+   On running the example above in sympy live, (1+1) is wrapped
```
Comment (aktech) [inline doc/src/tutorial/gotchas.rst]: See https://github.com/sympy/sympy/pull/10493#discussion_r51586846

**Comment by Arafatk** [comment]
@aktech Is it okay?

**Comment by aktech** [comment]
+1

**Pull request merged**